decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -265,7 +265,18 @@
         visibility: "all"
       ) do
         proposal = Decidim::Proposals::Proposal.new(params)
-        proposal.add_coauthor(participatory_space.organization)
+
+        coauthor = case n
+                   when 0
+                     Decidim::User.where(decidim_organization_id: participatory_space.decidim_organization_id).order(Arel.sql("RANDOM()")).first
+                   when 1
+                     Decidim::UserGroup.where(decidim_organization_id: participatory_space.decidim_organization_id).order(Arel.sql("RANDOM()")).first
+                   when 2
+                     Decidim::Meetings::Meeting.where(component: component).order(Arel.sql("RANDOM()")).first
```

Comment:
```suggestion
                     meeting_component = participatory_space.components.find_by(manifest_name: "meetings")
                     Decidim::Meetings::Meeting.where(component: meeting_component).order(Arel.sql("RANDOM()")).first
```
@codePau @Yeraldsb could you try to do this?
The point here is to find a meeting in the same participatory space, for that we can search for a component of the type "meetings" using the first line, then search a random meeting belonging to that component in the second line

Reply:
@microstudi Thanks for helping us, that solved it, now there are proposals created by meetings

Reply:
Great! please pass rubocop and let's see if this is done